she/recovery/dispatcher.py

```
@@ -0,0 +1,148 @@
+"""SHE P0.4 — dynamic dispatcher scaffold (pure ranking, no side effects).
+
+Ranks L0 recovery actions against a static capability table and an explicit
+authority set. MoneyBall scores are decision-support only and never grant
+execution. This module performs no network I/O and never mutates source.
+"""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Any, Mapping, Sequence
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Mapping`, `Sequence`

Import from `collections.abc`

(UP035)